<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookDialog</class>
 <widget class="QDialog" name="AddressBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="tools" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>850</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QToolButton" name="tbXinjian1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>90</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbDaoruDaoChu1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>90</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbShanchu1">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>5</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbShaxin1">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>5</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>5</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>200</width>
        <height>541</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>650</width>
        <height>540</height>
       </rect>
      </property>
      <widget class="QListWidget" name="lianxirensmsListWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>541</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>281</width>
         <height>541</height>
        </rect>
       </property>
       <widget class="QWidget" name="cardIcon" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>姓名:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>45</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>账号:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>分组:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbName">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lbUserName">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>45</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lbGroup">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QToolButton" name="tbCard">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>90</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tbEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>100</y>
          <width>60</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>手机号码:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbPhone">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pbSms">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <zorder>widget_3</zorder>
     <zorder>tools</zorder>
     <zorder>treeWidget</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="tools2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>850</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QToolButton" name="tbXinjian">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>90</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbDaoru">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbDaochu">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>5</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbShanchu">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>5</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>5</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="tbShaxin">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>5</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>650</width>
        <height>540</height>
       </rect>
      </property>
      <widget class="QListWidget" name="smsItemListWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbExplain">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>410</y>
         <width>641</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>(短信费用由运营商收取, 360手机助手不收取任何费用)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>521</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSend">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>430</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="smsListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>201</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="top" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QWidget" name="icon" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>360手机助手</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>top</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pbClose</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
